test/common.js

```diff
@@ -497,6 +497,12 @@ function fail(msg) {
 }
 exports.fail = fail;
 
+exports.mustNotCall = function(msg) {
+  return function dontCall() {
+    fail(msg || 'function should not have been called');
+  };
+};
+
```

Comment:
That will wait until the `exit` handler fires to fail. This fails immediately. Could be helpful for a test that otherwise will mysteriously time out because something is holding the event loop open indefinitely.